FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/FreeRTOS_ARP.c

```diff
@@ -86,8 +86,12 @@ static TickType_t xLastGratuitousARPTime = ( TickType_t ) 0;
 #endif /* ipconfigARP_USE_CLASH_DETECTION */
 
 /* Part of the Ethernet and ARP headers are always constant when sending an IPv4
-ARP packet.  This array defines the constant parts, allowing this part of the
-packet to be filled in using a simple memcpy() instead of individual writes. */
+ * ARP packet.  This array defines the constant parts, allowing this part of the
+ * packet to be filled in using a simple memcpy() instead of individual writes. */
+
+/* Rule has been relaxed since this is a configuration parameter. And this location provides
+ * more readability. */
+/* coverity[misra_c_2012_rule_8_9_violation] */
```

Comment:
I see no reason to suppress rule 8.9 here.  This is defined as `static` so it's not visible outside this module and it's referred to [only in a single location.](https://github.com/FreeRTOS/FreeRTOS/blob/master/FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/FreeRTOS_ARP.c#L627)  Readability will be improved by placing this definition in the block scope of `vARPGenerateRequestPacket()`, allowing you to delete lines 92-94 and eliminate this rule suppression.

Reply:
done...